site/cypress/e2e/Cypress-UI/navigation_page.spec.js

```diff
@@ -12,7 +12,7 @@ function validate_navigation_page_sections(sections) {
 describe('navigation page', () => {
     beforeEach(() => {
         cy.visit('/');
-        cy.wait(500);
+        // cy.wait(500);
```

Comment:
same about remove instead of comment

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/10562/commits/8107197e747bb610b3ae3ba2ba25d659559ebf33